src/sprints/sprints.service.ts

```diff
@@ -570,6 +571,12 @@ export class SprintsService {
             responsesArray,
         );
 
+        await canSubmitCheckin(
+            createCheckinForm.sprintId,
+            createCheckinForm.voyageTeamMemberId,
+            this.globalServices.validateOrGetDbItem,
```

Comment:
I tried to, but I'm not sure how to use it in individual ability files since there's no class there to use the constructor of. I think I was getting close to a solution working with an LLM, but then I ran out of free messages lol...

Reply:
I see. Maybe just leave it then unless you can come up with something soon. or leave a note there for future

Reply:
Yeah I'll just leave a note there for now, everything I tried ended up having something hacky about it. Seems like there should be a way to do dependency injection in situations like this, hmm...